substrate/primitives/consensus/grandpa/src/lib.rs

```diff
@@ -35,7 +35,7 @@ use sp_runtime::{
 };
 
 /// The log target to be used by client code.
-pub const CLIENT_LOG_TARGET: &str = "grandpa";
+pub const CLIENT_LOG_TARGET: &str = "finality_grandpa";
```

Comment:
Ah, yea it did a string replace. I also replaced the string with quotes, since there are feature gates that we call the same as the dependency.  
But i can run it again and print all the cases where it replaced an exact quoted string match.

Reply:
I checked the diff with `git diff -U0 $(git merge-base HEAD^ origin/master) -- ':!**/*.toml' | egrep '"(\w|_|-)+"'` and it seems like only a few cases where the string replace messed up.